packages/commonwealth/client/vite.config.ts

```diff
@@ -150,6 +154,31 @@ export default defineConfig(({ mode }) => {
     },
     resolve: {
       alias: [
+        {
+          // UNISWAP_WIDGET_HACK: 'jsbi' is needed by @uniswap pkg for pricing calculations, this is
+          // not documented by the uniswap pkg or atleast i couldn't find it.
+          // adding this here for internal uniswap widget import resolution
+          // see: https://github.com/Uniswap/sdk-core/issues/20 and
+          // https://github.com/Uniswap/widgets/issues/586#issuecomment-1777323003
+          // for more details
+          find: 'jsbi',
+          replacement: path.resolve(
+            __dirname,
+            '../node_modules/jsbi/dist/jsbi-cjs.js',
+          ),
+        },
+        {
+          // UNISWAP_WIDGET_HACK: needed by @uniswap pkg for path resolution
+          // see: https://github.com/Uniswap/widgets/issues/593#issuecomment-1777415001 for more details
+          find: '~@fontsource/ibm-plex-mono',
+          replacement: '@fontsource/ibm-plex-mono',
+        },
+        {
+          // UNISWAP_WIDGET_HACK: needed by @uniswap pkg for path resolution
+          // see: https://github.com/Uniswap/widgets/issues/593#issuecomment-1777415001 for more details
+          find: '~@fontsource/inter',
+          replacement: '@fontsource/inter',
+        },
```

Comment:
thats a weird thing that dependencies are not provided by the uniswap package. Wondering if this will work on remote environment correctly

Reply:
yeah. This is end up in the app build so any environment that uses the build will be fine.